nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/ElasticsearchRestProcessor.java

```diff
@@ -116,7 +117,7 @@ default String getQuery(FlowFile input, ProcessContext context, ProcessSession s
             session.exportTo(input, out);
             out.close();
 
-            retVal = out.toString();
+            retVal = out.toString(StandardCharsets.UTF_8);
```

Comment:
Whoops, looks like toString(Charset) was introduced in Java 10.

Reply:
Rings a bell from when I changed this originally actually... forgot to change the module settings in IntelliJ to check for JDK 8 compatability, d'oh.